packages/onboarding-ui/.storybook/preview.tsx

```diff
@@ -38,10 +38,9 @@ const getI18n = () => {
             awaitingConfirmation: {
               title: 'Awaiting confirmation',
               subtitle:
-                'We have sent you an email to emailname@email.com with a confirmation link. Please verify that the security code below matches the one in the email.',
-              code: 'Funny Tortoise In The Hat',
+                'We have sent you an email to {{emailAddress}} with a confirmation link. Please verify that the security code below matches the one in the email.',
               codeFallback:
-                'Didn’t receive email? <0>Resend</0> or <1>Change email</1>',
+                'Didn’t receive email? <1>Resend</1> or <3>Change email</3>',
```

Comment:
@tassoevan why?

Reply:
It allows us to show how the text is structured in the JSX too.

Reply:
got it, thank you!